test/artillery/performanceScript-users-userID.yaml

```diff
@@ -0,0 +1,47 @@
+# Thank you for trying serverless-artillery!
+# This default script is intended to get you started quickly.
+# There is a lot more that Artillery can do.
+# You can find great documentation of the possibilities at:
+# https://artillery.io/docs/
+config:
+  # this hostname will be used as a prefix for each URI in the flow unless a complete URI is specified
+  target: "http://nsc.functionsapp-team1.azurewebsites.net/api/"
+  phases:
+    
+      -duration: 50 #warmup
+       arrivalRate: 5
+        variables:
+         - user:
+              - "1"
+              - "2"
+              - "3"
+              - "5"
+              - "6"
+      
+     -duration: 200 #stress test
+      arrivalRate: 1
+      rampTo: 50
+      variables:
+         - user:
+              - "1"
+              - "2"
+              - "3"
+              - "5"
+              - "6"
+         - userName:
+              - "Jorden"
+              - "Aidan"
+              - "Macaulay"
+              - "Leroy"
+              - "Naomi"
+scenarios:
+  -
+    flow:
+
+      - get:
+          url: "/users/{{user}}"
+
+          #      - update:
+          # url: "/users/{{user}}"
+          # json:
+          #    firstname: "{{userName}}"
```

Comment:
can this be deleted?

Reply:
I was planning on working on this piece after we confirm that the normal test works